### ISSUE TITLE
Color scale customization

### ISSUE THREAD

**Issue by ud-klee** [StephanWagner/svgMap#15]
Like logarithmic scale? I tried to achieve that my overriding getColor() to translate the linear ratio to logarithmic and that fulfilled part of my needs. However, since the ratio is dependent on the data range i'm not able to apply a fixed scale. Perhaps make this an option and make the callback receive also the actual value in addition to just the ratio? Thanks.

**Comment by StephanWagner** [comment]
Yeah, good idea. I'm not too happy with the color options altogether and wanted to improve them anyway. Maybe I'll get around that while in quarantine.

**Comment by tijsgerwe** [comment]
Hi I'm pretty new to JavaScript.. and I am doing a project for school where we are building a site about traveling. We are using this JavaScript map to let the user know where he has been. Right now, all the information is loaded in the map; but the color of all the countries is black. When I try to change the color options nothing happens. I can see this post is about color so maybe someone can help me?

Thanks in Advance and thank you for letting us use this map!

**Comment by BenStigsen** [comment]
@tijsgerwe I know this is a very late reply to your school project.
When you create the map, you need to pass in the arguments. This is how I did it for a school project: 
![image](https://user-images.githubusercontent.com/24865450/111922769-06e2f300-8a9c-11eb-9d4a-1e0f578f2de5.png)
